# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libjpeg-turbo/hotfix-mips64.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libjpeg-turbo-3.1.0/simd/CMakeLists.txt.vanilla	2025-04-08 13:11:52.669670192 +0200
+++ libjpeg-turbo-3.1.0/simd/CMakeLists.txt	2025-04-08 13:12:24.035734988 +0200
@@ -442,7 +442,7 @@
 # MIPS64 (Intrinsics)
 ###############################################################################
 
-elseif(CPU_TYPE STREQUAL "loongson" OR CPU_TYPE MATCHES "^mips64")
+elseif(CPU_TYPE STREQUAL "loongson")
 
 set(CMAKE_REQUIRED_FLAGS -Wa,-mloongson-mmi,-mloongson-ext)
 
